scripts/dumps/dump_gene_set_from_core.pl

```diff
@@ -72,21 +79,26 @@ =head1 OPTIONS
 use Getopt::Long;
 use Pod::Usage;
 
-my ($dbname, $host, $port, $gene_set_dump_file, $help);
+my ($dbname, $host, $port, $gene_set_dump_file, $id_type, $help);
 
 GetOptions(
     'core-db|core_db=s' => \$dbname,
     'host=s'            => \$host,
     'port=s'            => \$port,
     'outfile=s'         => \$gene_set_dump_file,
+    'id_type=s'         => \$id_type,
     'h|help'            => \$help
 );
 
 pod2usage(1) if $help;
-unless ($dbname and $host and $port and $gene_set_dump_file) {
+unless ($dbname and $host and $port and $gene_set_dump_file and $id_type) {
```

Comment:
If the POD usage message would be acceptable for invalid values of `$id_type` (i.e. not `gene` or `protein`), it may be worthwhile to consider applying the `$id_type` regex already here, since an undefined value would not match the regex.